src/main/resources/fester_messages.xml

```diff
@@ -86,6 +86,7 @@
   <entry key="MFS-070">Unable to retrieve '{}' from the IIIF image server</entry>
   <entry key="MFS-071">Unexpected response code from the IIIF image server: {} [{}]</entry>
   <entry key="MFS-072">Looking up image width and height from: {}</entry>
+  <entry key="MFS-073">Setting image w/h to a questionable default value: 1 [item: {}]</entry>
```

Comment:
0

Reply:
I think it would be good to say _why_ we're setting the w/h to the questionable default (error(s) in the info.json).

Also, I'm not sure that it's necessarily true that both values would be set to `0` (which the error message could be construed to imply); is it possible to have generated an info.json that omits `width` but not `height`, or vice versa? If so, then the code in `ImageInfoLookup` could set one of those values to `0` and the other to something non-zero.

In either case, we have a non-comformant info.json, so IMO the error message should really be about that.

Reply:
Good points. I've updated the message text to better reflect what's happening.

Reply:
Looks good